modules/assim.batch/NEWS.md

```diff
@@ -0,0 +1,12 @@
+# PEcAn.assim.batch 1.7.2.9000
+
+## Breaking changes
+* In functions `gpeval` and `mcmc.GP`, parameter `splinefcns` has been renamed to `splinefuns` to match the spelling in `minimize.GP`, thereby also fixing several cases where the wrong name was passed between functions.
```

Comment:
@mdietze Does this name choice make sense to you? I'll flag the "several cases" below FYI.